packages/@okta/vuepress-site/docs/guides/style-the-widget/style-self-hosted/index.md

```diff
@@ -0,0 +1,241 @@
+---
+title: Style the self-hosted Sign-In Widget
+---
+
+This section discusses the customization options that you have when you are self-hosting the Sign-In Widget.
+
+> **Note:** You can try all of these customizations yourself using our [Live Widget](https://developer.okta.com/live-widget/).
+
+## Initial sign-in page
+
+You can modify the look of the initial sign-in page using parameters in the `config` section of the main Widget initialization block.
+
+![Screenshot of basic Okta Sign-In Widget](/img/widget_theming.png "Screenshot of basic Okta Sign-In Widget")
+
+### Logo
+
+```javascript
+var config = {
+  ...
+  logo: '/path/to/logo.png',
+  ...
+};
+
+var signIn = new OktaSignIn(config);
+```
+
+### Custom buttons
+
+You can add buttons below the **Sign In** button.
+
+```javascript
+var config = {
+  ...
+  customButtons: [
+    {
+      title: 'Click Me 1',
+      className: 'btn-customAuth',
+      click: function() {
+          // clicking on the button navigates to another page
+          window.location.href = 'http://www.example1.com';
+        }
+    },
+    {
+      title: 'Click Me 2',
+      className: 'btn-customAuth',
+      click: function() {
+          // clicking on the button navigates to another page
+          window.location.href = 'http://www.example2.com';
+      }
+    }
+  ],
+  ...
+};
+```
+
+### Links
+
+You can also change the **Help**, **Forgot Password**, and **Unlock** links, including both their text and URLs.
+
+```javascript
+var config = {
+  ...
+  helpLinks: {
+    help: 'https://example.com/help',
+    forgotPassword: 'https://example.com/forgot-password',
+    unlock: 'https://example.com/unlock-account',
+    custom: [
+      {
+        text: 'What is Okta?',
+        href: 'https://example.com/what-is-okta'
+      },
+      {
+        text: 'Example Portal',
+        href: 'https://example.com'
+      }
+    ]
+  },
+  ...
+};
+```
+
+## Modify the CSS
+
+In addition to the parameters in the Widget's `config`, you can also modify the CSS or override the default styles with your own.
+
+### Modify the existing theme
+
+If you want to add on top of the Okta theme, just edit [okta-theme.scss](https://github.com/okta/okta-signin-widget/blob/master/assets/sass/okta-theme.scss) and add any CSS to the bottom of the file.
+
+### CSS customization examples
+
+This section provides examples that you can use to make your own customizations to the widget.
+
+#### Background
+
+```css
```

Comment:
CSS code blocks are typically indented 2 spaces, not 4. I'd recommend we indent CSS code blocks with 2 spaces.